backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/NodeSizeProvider.java

```diff
@@ -25,12 +25,16 @@
  * Provides the Size to apply to a new node.
  *
  * @author fbarbin
+ * @author wpiers
  */
 public class NodeSizeProvider {
 
+    private static final int ELK_SIZE_DIFF = 24;
```

Comment:
Could you add a comment indicating what this magic number is, I personally do not remember.

Reply:
comment added in https://github.com/wpiers/sirius-components/commit/7b33800e71e2bcc8b2cb048f31b5ebb70ec8e681